### PR TITLE
Added back support for multiple tool list files

### PR DESCRIPTION
**Pull request by nuwang**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -11,9 +11,7 @@ galaxy_tools_galaxy_instance_url: 127.0.0.1:8080
 
 # A file containing all the tools to be installed. See `files/tool_list.yaml.sample`
 # file for more about the format requirements of this file. The file names must be unique.
-# TODO: Once Ansible 2.0 comes out, re-enable multiple files here
-# (see https://docs.ansible.com/ansible/playbooks_loops.html#loops-and-includes)
-galaxy_tools_tool_list: "tool_list.yaml"
+galaxy_tools_tool_list_files: [ "tool_list.yaml" ]
```
Comment (mvdbeek) [inline defaults/main.yml:14]: Can you change the comment to read "A list of yml files that list the tools to be installed" ?

**Comment by mvdbeek** [comment]
This looks nice, thanks a lot!

File: defaults/main.yml (hunk 1 above)
Comment (nuwang) [inline defaults/main.yml:14]: Thanks for catching that - fixed.

**Comment by bgruening** [comment]
:beers:

**Pull request merged**